packages/carbon-web-components/examples/codesandbox/basic/components/accordion/cdn.html

```diff
@@ -13,6 +13,8 @@
   <meta charset="UTF-8"/>
   <link rel="stylesheet"
         href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css"/>
+
+  <link rel="stylesheet" href="src/styles.scss" />
```

Comment:
does this work to link the cdn to the stylesheet?

Reply:
I thought it did! It doesn't 😭 I'll take them out